src/scalarstats.jl

```diff
@@ -294,6 +413,16 @@ end
 function entropy{T<:Real}(p::AbstractArray{T}, b::Real)
     return entropy(p) / log(b)
 end
+"""
+    entropy(p::AbstractArray{T}, b::Real)
```

Comment:
You need to document the default value for `b` (which corresponds to the method without this argument). Same for `crossentropy` and `kldivergence`.